module1/01_scrum.en.md

```diff
@@ -0,0 +1,19 @@
+<!-- .slide: data-background="#111111" -->
+
+# Teamwork - Scrum
+
+___
+
+##  Teamwork - Scrum
+
+* [Scrum meetings](https://github.com/coders-school/qa-sessions/raw/master/module1/ScrumFramework.pdf)
+
+* [Scrum cheat sheet](https://github.com/coders-school/qa-sessions/raw/master/module1/sciaga_scrum.pdf)
+
+___
+
+## Scrum - good practices
+
+* <!-- .element: class="fragment fade-in" --> Daily meetings - organized to synchronize workflow, see how the work is going and in particular who needs a helping hand because he stuck.
```

Comment:
```suggestion
* <!-- .element: class="fragment fade-in" --> Daily meetings - organized to synchronize workflow, see how the work is going and in particular who needs a helping hand because he is stuck.
```